### DISCUSSION TITLE
Getting Intelligent Dispatching Sessions

### DISCUSSION THREAD

**Discussion by cjl24** [BottlecapDave/HomeAssistant-OctopusEnergy#393] [General]
Not quite HA related. Although I am using the octopus intelligent HA integration to great effect I'm trying to figure out how you are accessing the intelligent dispatching sessions here as they don't seem to be part of the standard octopus API? I am trying to be able to access them just in a python console?

**Comment by BottlecapDave** [comment]
Hello. 

As per the api client class, they are accessed via the graphql version of the API. The query can be found at https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/custom_components/octopus_energy/api_client.py#L130
